src/ontoportal_client/api.py

```diff
@@ -110,7 +110,13 @@ def search(self, text: str, ontology: Optional[str] = None):
         params = {"q": text, "include": ["prefLabel"]}
         if ontology:
             params["ontologies"] = ontology
-        return self.get_json("/search", params)
+        page = 1
+        while page:
```

Comment:
this is a good place for the walrus operator

Reply:
Oh yeah, I forgot that exists! I gave it a try but what I ended up with felt very convoluted in order to only get the default `1` on the first iteration. Or maybe I'm not quite grasping what you had in mind.